interpreter/CMakeLists.txt

```diff
@@ -390,6 +382,13 @@ if (builtin_clang)
     CACHE STRING "Clang include directories.")
 endif()
 
+if (builtin_clang)
+  add_subdirectory(llvm/src/tools/clang EXCLUDE_FROM_ALL)
+else()
+  add_subdirectory(cling)
+endif()
+endif(builtin_llvm)
+
```

Comment:
I am wondering what's the observable change behavior of this change.

Reply:
I think `CLANG_INCLUDE_DIRS` needs to be set before doing `add_subdirectory(cling)`.

Reply:
I am a little confused, how did it work before?

Reply:
I have no idea, but I just double checked removing this change and [building clad fails]("https://gitlab.cern.ch/cburr/root-conda/-/jobs/2832171 ) (search for `Command failed: 2`).